packages/client/package.json

```diff
@@ -47,8 +48,8 @@
 		"@types/react": "^19.0.3",
 		"@types/react-dom": "^19.0.2",
 		"@types/semver": "^7.5.8",
-		"@typescript-eslint/eslint-plugin": "^8.19.1",
```

Comment:
this is not the same version as the other package versions, but i pushed a version to v2 that unifies them